projects/common/lib/components/address/address.component.html

```diff
@@ -38,6 +38,23 @@
       </button>
     </div>
   </div>
+  <!-- Address Validator -->
+  <div class="form-group col-sm-11 p-sm-0"
+       *ngIf="useAddressValidator">
+    <common-address-validator
+      name="street_{{objectId}}"
+      label="{{addrLabels.address1}}"
+      [(ngModel)]="addr.addressLine1"
+      (ngModelChange)="setStreetAddress($event)"
+      [required]="isRequired"
+      [disabled]="readOnlyFields.address"
+      [maxlength]="fieldMaxLengths.address"
+      [serviceUrl]="addressServiceUrl"
+      (select)="selectSuggestedAddress($event)"
+      [populateAddressOnSelect]="true"
+      commonValidateStreet>
+    </common-address-validator>
+  </div>
```

Comment:
I think this addition was due to a bad merge. The `common-address-validator` component is already used near the top of this file. The `common-address` component is used throughout MSP (contact pages), please make sure it's not rendering the component twice.

Reply:
Removed the duplication. Good eye